.github/workflows/images-latest.yml

```diff
@@ -51,34 +51,32 @@ jobs:
           - { name: python, ecosystem: pip }
           - { name: terraform, ecosystem: terraform }
     env:
-      TAG: ${{ github.sha }}
+      COMMIT_SHA: ${{ github.sha }}
       NAME: ${{ matrix.suite.name }}
       ECOSYSTEM: ${{ matrix.suite.ecosystem }}
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
 
-      - name: Build dependabot-updater image
+      - name: Build the dependabot-updater-<ecosystem> image
+        # despite the script input being $NAME, the resulting image is dependabot-updater-${ECOSYSTEM}
         run: script/build ${NAME}
 
-      - name: Log in to GHCR
+      - name: Tag the images with the SHA, `latest`, and the date version
         run: |
-          echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
+          docker tag "${UPDATER_IMAGE}${ECOSYSTEM}" "${UPDATER_IMAGE}${ECOSYSTEM}:$COMMIT_SHA"
+          docker tag "${UPDATER_IMAGE}${ECOSYSTEM}" "${UPDATER_IMAGE}${ECOSYSTEM}:latest"
+          docker tag "${UPDATER_IMAGE}${ECOSYSTEM}" "${UPDATER_IMAGE}${ECOSYSTEM}:${{ needs.date-version.outputs.date }}"
 
-      - name: Push branch image
```

Comment:
I suspect this allusion to branch name was a copy/paste leftover since we're _not_ on the branch, but rather on `main`...